docs/system/system-metrics.md

```diff
@@ -977,7 +978,12 @@ This metric is [recommended][MetricRecommended].
 
 | Name     | Instrument Type | Unit (UCUM) | Description    | Stability |
 | -------- | --------------- | ----------- | -------------- | --------- |
-| `system.filesystem.usage` | UpDownCounter | `By` |  | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
+| `system.filesystem.usage` | UpDownCounter | `By` |  [1] | ![Experimental](https://img.shields.io/badge/-experimental-blue) |
```

Comment:
brief is missing

Reply:
Brief added in https://github.com/open-telemetry/semantic-conventions/pull/1380/commits/536fbd8dd01542c0d9c619acab2ecc05de16c54d